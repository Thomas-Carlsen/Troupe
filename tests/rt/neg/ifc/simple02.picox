let val x = 10
    val y = 20
    val secret = mkSecret 42
in if secret > 0 then adv x else adv y
end
