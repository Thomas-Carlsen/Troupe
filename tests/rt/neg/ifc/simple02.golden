2019-03-07T12:08:24.898Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
[31mRuntime error in thread af73a050-f535-4d9a-b126-b48c741d36bf@{}%{}[39m
[31m>> Illegal flow in adv function:[39m
[31m |    pc: {#TOP}[39m
[31m | block: {#TOP}[39m
[31m | value: 10@{}%{}[39m
